CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ To install or update CocoaPods see this [guide](http://docs.cocoapods.org/guides
 
 To install release candidates run `[sudo] gem install cocoapods --pre`
 
+## Master
+
+##### Bug Fixes
+
+* Add missing `.cc` extension for C++ to `SOURCE_FILE_EXTENSIONS`.
```

Comment:
Perhaps this could be a bit more user friendly. The audience for this CHANGELOG is users, and not developers of CocoaPods. A reader most likely will have no idea what what `SOURCE_FILE_EXTENSIONS` is and it's use.

This should be more clear on what problem this change solves.